Removing Mentions of Make Your Mark from App (#446)
LibraryOfCongress/concordia: README.md

# Welcome to Concordia 

> Concordia is a platform developed by the Library of Congress (LOC) for crowdsourcing transcription and tagging of text in digitized images. The first iteration of Concordia was created by the LOC on crowd.loc.gov, launching in the autumn of 2018. 

Reviewer: “created by the LOC on crowd.loc.gov” reads a bit oddly to me. Maybe something like this?
```suggestion
Concordia is a platform developed by the Library of Congress (LOC) for crowdsourcing transcription and tagging of text in digitized images. The first iteration of Concordia was launched as crowd.loc.gov in the autumn of 2018. 
```
